### DISCUSSION TITLE
Can't Change Built-in App Icons

### DISCUSSION THREAD

**Discussion by alxnderclark** [elrumo/macOS_Big_Sur_icons_replacements#1509] [General]
Has anyone found a way to change the icons for iPhone Mirroring, Automator, etc.? There are icons available for download, but it seems like not a way to assign them.

**Comment by ZuBB** [comment]
apps that are part of macOS can't be altered in a regular way. you need to ~~go deeper (c)~~ disable SIP

**Comment by AClownfa** [comment]
Despite Apple's ongoing efforts to limit what you can do with the OS, [IconChamp](https://www.macenhance.com/iconchamp.html) will still allow you to change system icons. I've learned to avoid using it to manage non-system icons (stick with the old Get Info, Copy and Paste) and especially system extensions as things can get scrambled up; but if you just use it for system apps, it does the trick through Sequoia (not Tahoe compatible, unfortunately). Requires pro version for $10.
